Makefile

```diff
@@ -8,7 +8,7 @@ clean: Makefile.coq
 	rm -f Makefile.coq
 
 Makefile.coq: _CoqProject
-	$(COQBIN)/coq_makefile -f _CoqProject -o Makefile.coq
+	coq_makefile -f _CoqProject -o Makefile.coq
 
```

Comment:
Please keep the variable in place, this allows portability of the script. You should rather export it in your environment.

Reply:
> You should rather export it in your environment.

Disagreed. Why do you want to insist on COQBIN being exported? I have coq in my PATH (managed by opam). Setting COQBIN will just lead to inconsistencies down the line.

Notice that further up in the Makefile, you already rely on coqc being in the PATH.

Reply:
I found a snippet in ssreflect which makes it honor COQBIN if present, and fall back to the PATH otherwise.